includes/Commerce/Orders.php

```diff
@@ -96,7 +97,55 @@ public function update_local_order( Order $remote_order, \WC_Order $local_order
 	 */
 	public function update_local_orders() {
 
-		// TODO: implement
+		$page_id = facebook_for_woocommerce()->get_integration()->get_facebook_page_id();
+
+		try {
+
+			$response = facebook_for_woocommerce()->get_api()->get_new_orders( $page_id );
```

Comment:
We'll need to pass in the page access token for this API request, which we can now after implementing [this story](https://app.clubhouse.io/skyverge/story/62622/update-the-base-api-getter-to-accept-an-access-token)

Reply:
Fixed in https://github.com/facebookincubator/facebook-for-woocommerce/pull/1520/commits/5c72c5e8b4d3b62a9761eddd85f85824051f82e9